tools/klee/main.cpp

```diff
@@ -1481,8 +1667,50 @@ int main(int argc, char **argv, char **envp) {
     while (!seeds.empty()) {
       kTest_free(seeds.back());
       seeds.pop_back();
-    }
+    } */
+
+	// Fork server mode
+  int i = 0;
+  while(!interrupted) {
+   std::string newSeed = blocking_get_new_seed_file();
+   errs() << "read seed: " << newSeed << "\n";
+   errs() << "iteration " << i << "\n";
+   MemoryObject::counter = 0; // same MemoryObject ids for all runs for debugging memory layout
```

Comment:
good enough for testing. I wonder if it'd be cleaner to have a setter/getter function in MemoryObject to set/get this variable; instead of accessible like this.

Reply:
Definitely, but I think I will remove it altogether